src/modules/esl-mixin-element/ui/esl-mixin-attr.ts

```diff
@@ -0,0 +1,69 @@
+import {ESLMixinRegistry} from './esl-mixin-registry';
+import type {ESLMixinElement, ESLMixinElementInternal} from './esl-mixin-element';
+
+// Singleton cache for ESLMixinAttributesObserver instances
+const instances = new Map<string, ESLMixinAttributesObserver>();
+
+/**
+ * Internal {@link ESLMixinElement}s observedAttributes mutation listener.
+ * Creates a single instance per mixin type
+ * Ignores mixin primary attribute changes (they are observed by {@link ESLMixinRegistry} ootb)
```

Comment:
don't understand what _ootb_ means, please explain me

Reply:
out of the box, but not sure is that correct to write it lowercase